_tips/designing.ja.md

```diff
@@ -0,0 +1,669 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after "#".
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:"
+
+title: "ウェブ・アクセシビリティのためのデザイン – 始めるにあたってのヒント"
+title_html: "ウェブ・アクセシビリティのためのデザイン"
+nav_title: デザインのためのヒント
+lang: ja  # Change "en" to the translated-language shortcode
+last_updated: 2024-03-29   # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+resource:
+  ref: /tips/  # Do not change this
+
+navigation:
+  previous: /tips/writing/ # Do not change this
+  next: /tips/developing/ # Do not change this
+
+# translators: # remove from the beginning of this line and the lines below: "# " (the hash sign and the space)
+- name: "Jay Kishigami"   # Replace Jan Doe with translator name
+# - name: "Jan Doe"   # Replace Jan Doe with name, or delete this line if not multiple translators
+# contributors:
+# - name: "Jan Doe"   # Replace Jan Doe with contributor name, or delete this line if none
+# - name: "Jan Doe"   # Replace Jan Doe with name, or delete this line if not multiple contributors
+
+github:
+  repository: w3c/wai-quick-start
+  path: '_tips/designing.md'   # Add the language shortcode to the middle of the filename, for example: content/index.fr.md
+
+permalink: /tips/designing/   # Add the language shortcode to the end, with no slash at the end. For example /path/to/file/fr
```

Comment:
@jaykishigami Please add the language shortcode to the end of the permalink, to make the translation appear in the header.

See https://www.w3.org/WAI/about/translating/guides/new-translation/ for more guidance.